src/components/common/Profile.vue

```diff
@@ -82,13 +56,19 @@ export default {
     },
   },
   setup() {
-    function buttonFunction() {
-      console.log("Add modal here...");
-    }
+    const store = useStore();
+    const name = computed(() => store.getters.getName);
+    const lastname = computed(() => store.getters.getLastname);
+    const actualName = computed({
```

Comment:
to jest defaultowe zachowanie computeda, wiec nie musisz tego rodzielać na getter i setter
secundo, nigdzie nie używasz tej zmiennej więc nie wiem po co w ogóle ona jest